platforms/web/src/i18n/config.ts

```diff
@@ -25,19 +25,24 @@ export const DEFINED_LANGUAGES: LanguageDefinition[] = [
 const initI18n = async () => {
   const enabledLanguages = import.meta.env.APP_ENABLED_LANGUAGES?.split(',') || [];
   const defaultLanguage = import.meta.env.APP_DEFAULT_LANGUAGE || 'en';
+
   const supportedLanguages = filterSupportedLanguages(DEFINED_LANGUAGES, enabledLanguages);
 
   useConfigStore.setState({ supportedLanguages });
 
   if (!supportedLanguages.some(({ code }) => code === defaultLanguage)) {
     throw new Error(`The default language is not enabled: ${defaultLanguage}`);
+  } else {
+    // Set the default language on the HTML element
+    document.documentElement.lang = defaultLanguage;
   }
```

Comment:
If we resort to my previous suggestion, then I suggest applying the following code after initialisation of `i18n`. Then this code you written can be removed.

```js
if (i18next.language !== defaultLanguage) {
  i18next.changeLanguage(defaultLanguage);
}
```
This will cause our `languageChanged` event to fire.

This has the benefit of there is only one single point where `document.documentElement.lang` is set.

Some other notes about your approach (for the future in other occurrences):
1. Only comment code if it does not speak for itself or needs clarification. Otherwise it's redundant and we prefer not to comment code which is reasonably easy to understand.
2. An `else`-statement is not needed, because the code will stop executing after the error has been thrown. This is just a minor suggestion. It will make the code a little shorter and thus easier to read.

Reply:
Appreciate the suggestion, @langemike. I've incorporated your input and, following @ChristiaanScheermeijer's advice, opted for their streamlined approach. It requires fewer changes while achieving the same result.